src/Components/Common/Button.styles.scss

```diff
@@ -1,156 +1,54 @@
 @import "./../../styles/variables.scss";
 
-$black-opacity-12: rgba(0, 0, 0, .12);
-$black-opacity-14: rgba(0, 0, 0, .14);
-$black-opacity-20: rgba(0, 0, 0, .2);
-$gray-opacity-20: rgba(158, 158, 158, .2);
-$gray-opacity-40: rgba(158, 158, 158, .4);
-
 .btn {
-  display: inline-block;
-  font-weight: 400;
-  text-align: center;
-  vertical-align: middle;
-  cursor: pointer;
-  white-space: nowrap;
-  user-select: none;
-  padding: 11px $size-m * 2;
-  font-size: 1rem;
-  line-height: 1.5;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: $size-xs $size-xs-s;
+
+  font-size: $font-size-h5;
+  font-weight: 600;
   text-decoration: none;
-  text-shadow: none;
-  text-transform: none;
-  box-sizing: border-box;
-  margin: 0;
-  background-image: none;
-  background-color: $brand-blue;
-  color: $white-100;
-  border: 1px solid transparent;
-  border-radius: $border-radius-base;
+  cursor: pointer;
 
-  transition: background-color .2s;
+  border: $border;
+  border-radius: $border-radius-base;
+  background: $white-100;
+  color: $brand-primary-gray;
 
   &:hover {
-    text-decoration: none;
-  }
-
-  &:focus {
```

Comment:
we don't need focus anymore?

Reply:
Most of the css-classes were not used because all buttons were using the `button-material` class. I removed most css-classes to only apply our new design.